src/Performance.h

```diff
@@ -0,0 +1,60 @@
+#ifndef PERFORMANCE_H
+#define PERFORMANCE_H
+
+#include "Criteria.h"
+#include "Perf.h"
+#include <iostream>
+#include <string>
+#include <vector>
+
+class Performance {
+public:
+  /**
+   * Performance standard constructor
+   *
+   * @param id Id name of the performance (ex: name of alternative or profile)
+   * @param criteria Array of criterion to compute performance over
+   */
+  Performance(std::string id, Criteria &criteria);
+
+  /**
+   * Performance standard constructor
+   *
+   * @param id Id name of the performance (ex: name of alternative or profile)
+   * @param criteria Array of criterion to compute performance over
+   * @param performance Array of performance (float) to set
+   */
+  Performance(std::string id, Criteria &criteria, std::vector<float> &p);
```

Comment:
Is it a std::array or std::vector ? 
The arguments suggests vector and not arrays

Reply:
It's a std::vector, I'll update the doc